randovania/games/dread/json_data/Artaria.txt

```diff
@@ -456,7 +456,9 @@ Extra - asset_id: collision_camera_004
           Can Slide Underwater
           Before Artaria - Water Introduction Blob and Can Slide
   > Dock to EMMI Zone First Entrance (Lower)
-      Gravity Suit or After Artaria - Water Introduction Blob or Lay Cross Bomb or Use Spin Boost
+      Any of the following:
+          Gravity Suit or After Artaria - Water Introduction Blob or Lay Cross Bomb or Use Spin Boost
+          Speed Booster and Disabled Door Lock Randomizer
```

Comment:
I wouldnt put this in as trickless. put it as beginner. Lay Cross Bomb should also maybe be changed to Simple IBJ

Reply:
What kind of trick do you want here, speedbooster conservation or knowledge?

I brought up the cross bomb usage as something that can probably simply be removed, since the point of this connection is to skip breaking the blob, but since you have morph ball anyway, breaking the blob will be perfectly safe.